tools/cpp/windows_cc_configure.bzl

```diff
@@ -229,32 +229,38 @@ def _is_vs_2017_or_2019(vc_path):
     # C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\
     return vc_path.find("2017") != -1 or vc_path.find("2019") != -1
 
-def _find_vcvarsall_bat_script(repository_ctx, vc_path):
-    """Find vcvarsall.bat script. Doesn't %-escape the result."""
+def _find_vcvars_bat_script(repository_ctx, vc_path):
+    """Find batch script to set up environment variables for VC. Doesn't %-escape the result."""
     if _is_vs_2017_or_2019(vc_path):
-        vcvarsall = vc_path + "\\Auxiliary\\Build\\VCVARSALL.BAT"
+        vcvars_script = vc_path + "\\Auxiliary\\Build\\VCVARSALL.BAT"
```

Comment:
Is there a vcvars64.bat in this case, and if so, shall we use it here too? We only support 64 bit anyway.

Reply:
That's a good idea! 
Fun fact: in VC 2015, `VCVARSALL.BAT` calls platform specific scripts (`VCVARS64.bat`, ect), in VC 2017 & 2019, platform specific scripts call `VCVARSALL.BAT`...

Reply:
Nice.